example_simple_regression.py

```diff
@@ -0,0 +1,70 @@
+"""This file is used to create and compare two models on a particular dataset.
+It provides examples of reading from both csv and SQL Server. Note that this
+example can be run as-is after installing healthcare.ai. After you have
+found that one of the models works well on your data, move to Example2
+"""
+import pandas as pd
+import time
+
+import healthcareai.common.file_io_utilities as io
+import healthcareai.pipelines.data_preparation as pipelines
```

Comment:
This is also potentially confusing to import a module and name in a way that shadows another module name (even if it's one of our own).

Here's how I recommend doing it:
```
from healthcareai.pipelines import data_preparation
```
I actually don't recommend shortening this here either.  If it were sprinkled throughout the code, there's an argument to import it as something shorter.  However, since this is only used once below - it's OK to spell it out in that instance.